tests/functional/karate_tests/src/test/java/com/karate/openwhisk/functions/hello-world.js

```diff
@@ -0,0 +1,8 @@
+function() {
+    // Licensed to the Apache Software Foundation (ASF) under one or more contributor
+    // license agreements; and to You under the Apache License, Version 2.0.
```

Comment:
move license to top of file

Reply:
Karate Doesn't allow using the comments outside the function in a JS file